x230/README.md

```diff
@@ -263,3 +266,7 @@ Mainly a tiny binary used by the Ethernet card and the Intel Management Engine.
 * `git clone https://github.com/merge/skulls`
 * rename one of the included config files to `config-xxxxxxxxxx`.
 * The x230 directory's `./build.sh` should produce the exact corresponding release image file.
+
+## Further reading / more pictures
+* [Flashing the X230 with coreboot while slaying Intel-ME](https://steemit.com/tutorial/@joeyd/run-don-t-walk-from-the-blob)
+* [Step by Step easy guide to flashing Coreboot (X230, but should work for others too)](https://www.reddit.com/r/thinkpad/comments/4zrmf8/step_by_step_easy_guide_to_flashing_coreboot_x230/)
```

Comment:
let's avoid linking to tutorials and add everything we need to the README.

Reply:
I found the pictures in the steemit-tutorial. But not strictly required.